doc/version_history.rst

```diff
@@ -5,6 +5,10 @@
 ===============
 Version History
 ===============
+v1.25.0
```

Comment:
Can you add an empty line before this line?

Reply:
I think I messed up the spaces with the rebase conflict.